docs/en/latest/references/apisix_route_v2.md

```diff
@@ -72,6 +72,9 @@ The table below describes each of the attributes in the spec. The fields `apiVer
 | http[].authentication.type           | string             | Type of authentication. Set to one of `basicAuth` or `keyAuth`.                                                                                                                           |
 | http[].authentication.keyAuth        | object             | Unique key for a [Consumer](https://apisix.apache.org/docs/apisix/terminology/consumer/).                                                                                                 |
 | http[].authentication.keyAuth.header | string             | Header to get the key from.                                                                                                                                                               |
+| http[].upstreams                     | array              | Upstreams refer to ApisixUpstream CRD.                                                                                                                                                    |
+| http[].upstreams[].name              | string             | Upstream name refer to ApisixUpstream CRD.                                                                                                                                                |
+| http[].upstreams[].weight            | integer            | The proportion of selections for each upstream.                                                                                                                                           |
```

Comment:
```suggestion
| http[].upstreams                     | array              | List of `ApisixUpstream` resources.                                                                                                                                                    |
| http[].upstreams[].name              | string             | Name of the `ApisixUpstream` resource.                                                                                                                                                |
| http[].upstreams[].weight            | integer            | Weight with which to split traffic to the upstream.                                                                                                                                           |
```

Reply:
@pottekkat Thank you for your quick review. Applied it right away.